hack/jenkins/e2e-image/Makefile

```diff
@@ -15,6 +15,8 @@
 IMG = gcr.io/google-containers/kubekins-e2e
 TAG = $(shell date +v%Y%m%d)
 
+all: push
```

Comment:
well, used to. it looks like you changed `kubekins-test` to only build?

I've updated this one to only build to be consistent.